Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -220,7 +220,7 @@ echo disk > /sys/power/state
 
 		# Check the system log if it shows Power Management log
 		"hibernation entry", "hibernation exit" | ForEach-Object  {
-			$pm_log_filter = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "dmesg | grep -i '$_'" -ignoreLinuxExitCode:$true
+			$pm_log_filter = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "cat /var/log/syslog | grep -i '$_'" -ignoreLinuxExitCode:$true
```

Comment:
some distro doesn't contain syslog, but ubuntu has

Reply:
Fixed in 8aa770200caaa33e45c99935b255d3631955c757